plugin/pkg/admission/limitranger/admission_test.go

```diff
@@ -743,8 +743,8 @@ func TestLimitRangerIgnoresSubresource(t *testing.T) {
 		t.Errorf("Expected an error since the pod did not specify resource limits in its create call")
 	}
 	err = handler.Validate(context.TODO(), admission.NewAttributesRecord(&testPod, nil, api.Kind("Pod").WithVersion("version"), limitRange.Namespace, "testPod", api.Resource("pods").WithVersion("version"), "", admission.Update, &metav1.UpdateOptions{}, false, nil), nil)
-	if err != nil {
-		t.Errorf("Expected not to call limitranger actions on pod updates")
+	if err == nil {
```

Comment:
why `err == nil` but print `t.Errorf(`

Reply:
Test cases are needed to verify whether the code we modified complies with the business logic. It is not just for passing the test.

Reply:
to allow pod update , previous we do not allow  update. 
what kind of test cases ? can you show me  how to write? 
@googs1025

Reply:
First, you need to modify the unit test code itself, not just to pass the tests.... Second, since you've made changes, you should incrementally cover the logic of your changes to ensure the code's rationality.

Reply:
If you encounter difficulties during the process, perhaps you could consider handing over the issue to someone else.

Reply:
@mouuii :)